completion/openqa-cli-completion.zsh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env zsh
+
+_openqa_cli_completions() {
+    local curcontext="$curcontext" state line
+    typeset -A opt_args
+    local -a subcommands
+    local -a main_options
+    local -a output_options
+    local -a api_options
+    local -a archive_options
+    local -a monitor_options
+    local -a schedule_options
+    subcommands=("api" "archive" "monitor" "schedule")
+    main_options=(
+        '--apibase[Base URL for the API]'
+        '--apikey[API key for authentication]'
+        '--apisecret[API secret for authentication]'
+        '--help[Show help message]'
```

Comment:
```suggestion
        '--help[Display help with available subcommands and options]'
```

Reply:
The options and its descriptions are copied from https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/CLI/api.pm#L115 , https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/CLI/schedule.pm#L55 etc.
Apparently for some the content was changed, or a mistake happened when copying & pasting.

We discussed already above that we want some kind of deduplication first, so that we don't have that content in two more places.